pkg/helm/client.go

```diff
@@ -140,14 +142,24 @@ func (h CliClient) Install(ctx context.Context, chart, version, release, namespa
 	if wait {
 		cmd = append(cmd, "--wait")
 	}
-
+	if !dryRun {
+		out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
+		if err != nil {
+			log.Error().Print("Error installing helm chart", field.M{"output": out})
+			return "", err
+		}
+		log.Debug().Print("Result", field.M{"output": out})
```

Comment:
If we get a log just saying `Result something`. I don't think we would really  know what exactly this result is. Let's improve this message to say what this result is.

Reply:
Done